### DISCUSSION TITLE
Databricks module

### DISCUSSION THREAD

**Discussion by Ulveno** [Azure/ResourceModules#3088] [Ideas]
Hello, 

Awesome work with all the modules. I was testing the Databricks module but it looks like there are missing properties params.

**Comment by eriqua** [comment]
Hi @Ulveno, thanks for the feedback and for opening this discussion. Sorry for the late reply. Discussions are currently not monitored promptly and are planned to be disabled in favor of the new issue template [Questions/Feedback].

I'm converting this discussion to an issue in order for the team to be able to triage it.

Please refer to issue #3144 going forward.
